tests/test_utils/test_transactiontestcase.py

```diff
@@ -1,7 +1,27 @@
+
+from django.db import connections
 from django.test import TransactionTestCase, mock
 
+from .models import Car
+
+
+class TestSerializedContentMockMixin(object):
+    initial_data_migration = '[]'
+    _connections_test_serialized_content = {}
+
+    def _pre_setup(self):
```

Comment:
I was sharing the same feeling, but I'm testing something that is related to the testing environment, which is a kind of special case.

By overwriting these methods, I want to be sure to control the `_test_serialized_contents` data during the whole test. It is important to mock them also in `TestSerializedRollbackInhibitsPostMigrate` test.

If we don't mock it, the logic here tries to load some data from previous django test suite serialized contents (because the test suite has some initial data migrations), which adding some dependencies between tests in the test suite.

This mock should be done every time we want to test something with `TransactionTestCase` and `serialized_rollback = True` to avoid test dependencies. Maybe some documentation to explain it would be nice.